.github/workflows/reusable_build.yml

```diff
@@ -58,10 +61,19 @@ jobs:
         run: |
           cat >> "$GITHUB_ENV" << 'EOF'
           ${{inputs.additional_envs}}
-          ${{secrets.secret_envs}}
           DOCKER_TAG<<DOCKER_JSON
           ${{ toJson(fromJson(inputs.data).docker_data.images) }}
           DOCKER_JSON
+          ROBOT_CLICKHOUSE_SSH_KEY<<RCSK
```

Comment:
Could you check if it will be empty in the case of `PRIVATE_CI_DB_PASSWORD = os.getenv("PRIVATE_CI_DB_PASSWORD", "")`?

Reply:
it's empty